enzy_htp/_interface/amber_interface.py

```diff
@@ -1251,6 +1249,42 @@ def get_amber_atom_index(self, atoms: List[Atom]) -> List[int]:
         aid_mapper = self.get_amber_index_mapper(stru)
         return [aid_mapper["atom"][at] for at in atoms]
 
+
+    def rename_atoms(self, stru: Structure) -> None:
```

Comment:
The design for atom names is that:
- we use a naming conventions as a standard in EnzyHTP
- file parsers needs to parse non-standard names to standard ones when reading and vice versa

Reply:
In this case, we should not change atom names in Structure() (it is also dangerous as some functions relies on it to figure out the topology.). This function should be in e.g.: pdb_io and we can add options for the PDB style of input or output.

Reply:
This function is the result of some problems I have come across. Namely, the default naming scheme in EnzyHTP is essentially that of the Amber force field. As a result, things from Rosetta have to have special treatment before they are ready for MD. If there is a current way to do it, let me know. I guess I can try to use the original naming on the output from reactive docking but in general I don't think EnzyHTP is currently robust with respect to atom naming and conversion between packages.

Reply:
What do you want me to do with this?

Reply:
The goal is to make a new feature in PDBParser() (e.g.: PDBParser().get_structure(format="rosetta") and PDBParser().get_file_str(format="amber")) to canonicalize the atom naming. (e.g.: there will be an atom name mapper for each PDB format variant) We can add it after line 120 that built the Atom()s or after built the Structure() and in _write_pdb_atom(). 

We can have it in this PR or mark a TODO next to this function and make it in the next PR. ( I can help work on it )

Reply:
Yes I like this design. I think ultimately we need a function that maps atom names that is then used in the parser, but can be used elsewhere also. If we had this function it could be useful to other workflows (i.e. `dock_reactants()` since you have to use Rosetta naming in that scheme.

Reply:
Cool! I will make TODOs both here and pdb_io